### ISSUE TITLE
Additional input on Canada, Canada/Ontario, Canada/Manitoba, Canada/British Columbia

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-policies-prototype#339]
From direct e-mail to Shawn:

I was scanning through the "Web Accessibility Laws & Policies" and notice for the Canadian laws, there is no mention of the "Accessibility for Ontarian's with Disabilities Act" (AODA). In Canada, accessibility laws are currently primarily provincial jurisdiction. While Ontario has taken the lead, British Columbia and Manitoba are now following them with similar laws.

I think AODA in particular should be mentioned. It's acting as a model for many other jurisdications. It was enacted in 2005, and has evolved into the Integrated Accessibility Standards Regulation (IASR). While the regulations go well beyond Digital/Web accessibility, the relevant standard is the Information and Communications Standard, which adopts WCAG 2 almost verbatim (less 1.2.4, 1.2.5). The law applies to all organization with 50 or more employees (both public and private).

AODA https://www.ontario.ca/laws/statute/05a11

IASR https://www.ontario.ca/laws/regulation/110191

Manitoba http://www.accessibilitymb.ca/law.html

British Columbia https://www2.gov.bc.ca/assets/gov/government/about-the-bc-government/accessible-bc/accessibility-2024/docs/building-a-better-bc.pdf

There is also a Canadian federal legislation in the works, in addition to the Canadian Human Rights Act, and Standard on Web Accessibility (as opposed to the Policy on Communications and Federal Identity listed in the W3C laws & policies page).

https://www.canada.ca/en/employment-social-development/programs/accessible-people-disabilities/act-plain-language-summary.html

https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=23601

---

Main issue: https://github.com/w3c/wai-policies-prototype/issues/285

Related data:  https://github.com/w3c/wai-policies-prototype/issues/327 , https://github.com/w3c/wai-policies-prototype/issues/272 , https://github.com/w3c/wai-policies-prototype/issues/271
